tests/suites/test_suite_x509write.function

```diff
@@ -313,7 +313,7 @@ void x509_crt_check( char *subject_key_file, char *subject_pwd,
     if( pk_wrap == 2 )
         TEST_ASSERT( mbedtls_pk_get_type( &issuer_key ) == MBEDTLS_PK_OPAQUE );
 
-    TEST_ASSERT( mbedtls_test_read_mpi( &serial, 10, serial_str ) == 0 );
+    TEST_ASSERT( mbedtls_test_read_mpi( &serial, serial_str ) == 0 );
```

Comment:
so which `.data` file did this require modification to?

Reply:
The test cases for this function all use the same `serial_str` value of "1", so no `.data` change is needed as the representation is the same for either radix.